### PR TITLE
Update main.html

### PR DESCRIPTION
**Pull request by M0N57R0517Y**
Added new rotation list, extra slot for rotation scrambles, fixed lack of GEN explanations at bottom, added new rules.

### CODE REVIEW

**Comment by M0N57R0517Y** [comment]
OK I think I did this right.

**Comment by icyflame** [comment]
Thanks, let me have a look and merge this. :+1:

File: main.html [hunk 1]
```diff
@@ -189,21 +252,29 @@
 
 > 1.
 
+**2GEN:**(in this mode of solving, you can only do <R,U>, which has the moves R, R', R2, U, U', U2)
```
Comment (icyflame) [inline main.html:255]: Here the open and close angular brackets are not shown, because they are interpreted as HTML tags. I am fixing this.

Comment (M0N57R0517Y) [inline main.html:255]: Yup, that's why they weren't showing up in the old markup.  Thanks for seeing that.

Comment (icyflame) [inline main.html:255]: It is fixed now, I used `&lt;` and `&gt;` :smile:

**Pull request merged**